insights/parsers/rpm_v_packages.py

```diff
@@ -1,6 +1,6 @@
 """
-RpmVPackages - command ``/bin/rpm -V coreutils procps procps-ng shadow-utils passwd sudo``
-==========================================================================================
+RpmVPackages - command ``/bin/rpm -V coreutils procps procps-ng shadow-utils passwd sudo chrony``
```

Comment:
How about just change it as 

```
RpmVPackages - command ``/bin/rpm -V <package names>``
```

So, we do not need to update it whenever adding new packages.

Reply:
@xiangce Sure. My concern was that the user will have no idea which packages are verified. He will always have to refer the specs. To void this, I have include the list in the docstring.

Reply:
Done.